src/styles/main.scss

```
@@ -1,3 +1,58 @@
+$calendar-item-length: 100px;
+$calendar-length: ($calendar-item-length * 7 + 20);
+
 body {
   margin: 0;
+  height: 100vh;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  font-family: Arial, Helvetica, sans-serif;
+  font-size: 30px;
+}
+
+.calendar {
+  max-width: #{$calendar-length};
+  padding: 10px;
+  display: flex;
+  align-items: center;
+  justify-content: flex-start;
+  gap: 1px;
+  flex-wrap: wrap;
+  &__item {
```

```suggestion
  flex-wrap: wrap;

  &__item {
```

to increase readability
fix everywhere